markdown/3.x/en/documentation/include-scripts.md

```diff
@@ -24,16 +24,19 @@ Depending on the browser's page loading time you might encounter ugly flashes of
 ```html
 <ff-record-list unresolved></ff-record-list>
 ```
-Also add the following CSS rule to the top of the page before the appearance of FACT-Finder Web Components:
+Also add suitable CSS rule to the top of the page before the appearance of FACT-Finder Web Components.
+
+The following CSS snippet hides element marked as unresolved (as well as its descendants) in the document:
 
 ```html
 <head>
     <style>
         [unresolved] {
-            visibility: hidden;
+            display: none;
```

Comment:
I'm not sure whether `visibility: hidden` isn't the generally more suitable style rule. `display: none` should only be useful for elements that don't usually take up space in the regular document flow. Like ff-suggest's popup.

I would like to see a second rule here. Like:
```html
ff-suggest[unresolved] {
    display: none;
}
```
So it becomes clear that it might make sense to distinguish methods of hiding unresolved elements.

Reply:
I think `display: none` is fine, until you set fixed size of the component (width and height). What is the benefit of `visibility: hidden` if you don't know its size? Let's say we have `ff-asn` with some custom templates. It takes some small place, but when rendered it will expand its area. Having something under filters, you will a blank space over it and then suddenly filters appear. In such case user can add some spinner styles until the component shows up. Can you find some practical use of `visibility: hidden` in the FF Web Components? Besides, they render quickly (event when waiting for data from FACT-Finder they mostly render with empty initial data and appear after retrieving something from the server).